pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -257,6 +257,13 @@ const (
 )
 
 type cpuAccumulator struct {
+	// `logger` is the logger instance the cpuAccumulator is going to be use. We store internally
+	// because the expected lifetime of a cpuAccumulator object is fully contained in a
+	// `takeByTopology` call. Hence there's no functional difference between storing it or
+	// getting repeatedly from the calls. Furthermore, this way we can keep the API surface smaller
+	// (less parameters).
+	logger logr.Logger
```

Comment:
I'd suggest to use klog.Logger instead. logr is an indirect dependency and theoretically can be changed.

Reply:
sure, will change in the next upload